python/nano/src/bigdl/nano/deps/ipex/ipex_inference_bf16_model.py

```diff
@@ -94,17 +96,25 @@ def status(self):
         return status
 
     @staticmethod
-    def _load(path, model, inplace=False):
+    def _load(path, model, inplace=False, input_sample=None):
```

Comment:
A quick thought, we have fairly vague definition what should be done in `_load` and what should be done in `__init__` when users call `InferenceOptimizer.load`, we may have a better (at least internal) understanding for this.

Reply:
we may update here later like give more detailed comment.